app/scripts/controllers/newOverview.js

```diff
@@ -1263,6 +1263,7 @@ function OverviewController($scope,
         resource: 'bindings'
       }, context, function(bindings) {
         state.bindings = bindings.by('metadata.name');
+        overview.bindingsByInstanceRef = _.indexBy(state.bindings, 'spec.instanceRef.name');
```

Comment:
Heh. Was messing with that before I read this.  Getting only 1 item every time was confusing 😄